app/views/asp/schoolings/index.html.haml

```diff
@@ -0,0 +1,19 @@
+.fr-mb-3w
+  = form_with method: :get, url: asp_schoolings_path do |form|
+    .fr-search-bar.fr-search-bar--lg.fr-col-md-8#search
+      = form.label :search, "Numéro de décision d'attribution, de dossier ou de prestation dossier", class: "fr-label"
+      = form.text_field :search, placeholder: "Numéro de décision d'attribution, de dossier ou de prestation dossier", value: @search, class: "fr-input"
+      = form.submit "Rechercher", class: "fr-btn"
```

Comment:
faut que ce soit un `%button` pour que ce soit stylisé correctement par le DSFR (c.f layout mobile/small)

Reply:
Est-ce que ça mérite de refaire ou est-ce que c'est suffisant pour pousser rapidement l'interface ?

Reply:
ça mérite de refaire

Reply:
ok, i'm on it